src/core/src/pso.rs

```diff
@@ -208,7 +208,7 @@ impl Descriptor {
 }
 
 /// A complete set of vertex buffers to be used for vertex import in PSO.
-#[derive(Copy, Clone, Debug)]
+#[derive(Copy, Clone, Debug, PartialEq)]
```

Comment:
I'm afraid so because it needs to be able to compare the existing `VertexBufferSet` against the one it has cached.  It can probably be worked around by just storing the *contents* of the `VertexBufferSet`, but that seems a little redundant.